Assets/_BForBoss/_Core/Scenes/AdditiveHUDScene.unity

```diff
@@ -0,0 +1,366 @@
+%YAML 1.1
```

Comment:
Can you remove the `EventSystem` object from this scene

Reply:
Updated